src/components/common/Extensions/UserHub/UserHub.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+import { AnimatePresence, motion } from 'framer-motion';
+import { useIntl } from 'react-intl';
+import { useMobile } from '~hooks';
+import ReputationTab from './partials/ReputationTab';
+import StakesTab from './partials/StakesTab';
+import TransactionsTab from './partials/TransactionsTab';
+import { transactionsItems } from '~common/Extensions/UserHub/partials/TransactionsTab/consts';
+import Button from '~shared/Extensions/Button';
+import Icon from '~shared/Icon';
+import styles from './UserHub.module.css';
+import { tabList } from './consts';
+import UserHubMobile from './UserHubMobile';
+
+export const displayName = 'common.Extensions.UserHub.partials.UserHub';
+
+const UserHub = () => {
+  const isMobile = useMobile();
+  const { formatMessage } = useIntl();
+  const [selectedTab, setSelectedTab] = useState(0);
+
+  const handleChange = (selectedOption: number) => {
+    setSelectedTab(selectedOption);
+  };
+
+  return (
+    <div className={`${isMobile ? 'flex-col' : 'flex-row'} flex`}>
```

Comment:
You don't have to use `flex-row` property because it is used by flex by default. 

Maybe better is to use: `<div className={clsx('flex', { 'flex-col': isMobile})}/>`